implementations/rust/ockam/ockam_identity/tests/channel.rs

```diff
@@ -1049,3 +1049,78 @@ async fn test_channel_delete_ephemeral_keys(ctx: &mut Context) -> Result<()> {
 
     Ok(())
 }
+
+#[allow(non_snake_case)]
+#[ockam_macros::test]
+async fn should_stop_encryptor__and__decryptor__in__secure_channel(
```

Comment:
We don't have a strong enforcement of names in our unit tests at the moment but my preference would be to

 - start a test function with `test` (we have this in many places)
 - don't try to use `__` in the function names so that: `#[allow(non_snake_case)]` is not necessary. I think that in most cases the name reads just as fine

This is just my preference :-)